### ISSUE TITLE
Videos not showing on the site after upload

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8205]
Since yesterdays update users cannot upload anymore. No new uploads show in the admin panel. The uploads are there in the users video panel. No logs in php log, apache2 log or avideo.log. Users are doing my nut in complaining they cant upload.

All 3 encoders go through the motions and direct upload without error but nothing shows on the site. No uploads for 20 hours now.

cheers

**Comment by GreenDragonTavern** [comment]
@para2022 - The videos are probably there, just not where they belong in descending order by date. I've had to scroll through all videos just to find a couple most recent uploads. Did find them, just not at the top. It affects **Date Added** too on Front Page. Again, I found the videos, but not at the top where they belong. Definitely a bug.

**Comment by para2022** [comment]
It is othing to do with that. As videos would show in the admin panel when users are encoding their own vids? Well that isnt happening and they are nowhere to be found.

**Comment by para2022** [comment]
why is screwed up? it was working absolutely fine. Now my sites vids are all over the place.

**Comment by para2022** [comment]
Yes ive noticed thats whats screwed my site up. Seems a useless function to me. Really need a fix for this as my users are kicking off as usual as it all looks a mess. I have last weeks vids first and the recent ones burried on page 10 lol

**Comment by GreenDragonTavern** [comment]
@para2022 - To fix, My Videos, click the **Sort Videos*, and delete all videos in the list. Then click add (make sure it's blank), and then save. Life should return to normal 😁

**Comment by DanielnetoDotCom** [comment]
Get the last update and delete all the videos in the sort menu then click on save

![chrome-capture-2023-6-20](https://github.com/WWBN/AVideo/assets/18009293/681a15a2-b130-4d3c-bfbf-49ce0990878e)

**Comment by walt93** [comment]
Ohhhhh is this what I was tripping over between the NULL/0 value for 'order'? Seems messy.

How is it supposed to work?

Daniel, it would be groovy and delicious if you could get into the habit of implementing a "kill switch" when you put new functionality in like this. Default it to off. Make it opt-in...

**Comment by para2022** [comment]
> Ohhhhh is this what I was tripping over between the NULL/0 value for 'order'? Seems messy.
> 
> How is it supposed to work?
> 
> Daniel, it would be groovy and delicious if you could get into the habit of implementing a "kill switch" when you put new functionality in like this. Default it to off. Make it opt-in...

Agreed, please set things to 'off' as default so we are not caught off guard and panicking LMAO

**Comment by para2022** [comment]
> Get the last update and delete all the videos in the sort menu then click on save
> 
> ![chrome-capture-2023-6-20](https://user-images.githubusercontent.com/18009293/254974704-681a15a2-b130-4d3c-bfbf-49ce0990878e.gif)

Ok will do. Thanks Dan

**Comment by para2022** [comment]
> @para2022 - To fix, My Videos, click the *_Sort Videos_, and delete all videos in the list. Then click add (make sure it's blank), and then save. Life should return to normal 😁

Thanks pal

**Comment by para2022** [comment]
I get an error popup if i delete them and save

**Comment by DanielnetoDotCom** [comment]
what error?
did you just make the git pull?

**Comment by para2022** [comment]
its a pop up window with a red 'error' box

Yes i made a git pull

**Comment by DanielnetoDotCom** [comment]
please make a git pull again, if it persists please check your logs

**Comment by para2022** [comment]
Ok.
Have you done something to the css? the side bar isnt loading properly now. The page loads with all the side panel links (not the side bar itself) then the links go away and the sidebar loads but it stays out rather than how it was before.

https://ourtube.co.uk

**Comment by para2022** [comment]
Oh i cleared the cache and restarted apache it made no difference.

**Comment by DanielnetoDotCom** [comment]
please try this update

**Comment by JoshWho** [comment]
So is this fixed?

**Comment by JoshWho** [comment]
![ddde1189-58f0-4c61-8cf4-c6b9642c553e](https://github.com/WWBN/AVideo/assets/39392449/f53b9b7a-00a4-4e49-97fa-94613362211f)

I see people uploading but this never changes

**Comment by DanielnetoDotCom** [comment]
there is something in the sort menu?

https://github.com/WWBN/AVideo/issues/8205#issuecomment-1644397875

what happen if you clear the cache?

**Comment by JoshWho** [comment]
It is cleared still just putting videos behind older videos

**Comment by DanielnetoDotCom** [comment]
![image](https://github.com/WWBN/AVideo/assets/18009293/6c2edc7b-b593-49a9-80f0-392327e5b4c8)

**Comment by JoshWho** [comment]
So every user on my platform has to do this?

**Comment by JoshWho** [comment]
Ok i figured it out. All good now
